worlds/sc2wol/LogicMixin.py

```diff
@@ -1,5 +1,5 @@
 from BaseClasses import MultiWorld
-from ..AutoWorld import LogicMixin
```

Comment:
Should probably update those imports in the world api.md and all worlds, so it's less work to make it apworld-compatible.